### ISSUE TITLE
Design Manual home page design updates

### ISSUE THREAD

**Issue by mebates** [cfpb/design-manual#275]
Related to #273, is the type size for the intro paragraph in the Design Manual too large? Originally, the design was for much shorter content, so it looked nicer paired with the illustration. Now it's just a huge block with lots of white space below the illustration. I would suggest moving it to the H4 (18px) at either Regular or Medium. 

![screen shot 2015-02-03 at 2 43 00 pm](https://cloud.githubusercontent.com/assets/6223901/6027859/fc58fc82-abb2-11e4-99bb-54dc0453b896.png)

**Comment by Dnpizarro** [comment]
@mebates I would agree that it feels like too much text, especially at that size. Could I make a content suggestion with regards to that first paragraph. I think the 2nd paragraph really introduces the DM well, maybe if we eliminated the first paragraph and just kept the 2nd one I think that could solve part of the issue.

**Comment by stephanieosan** [comment]
I think the first paragraph gives context about why it's important and worthwhile to the CFPB and the American public that the manual exists, though. It's the intro you need if you're not familiar with the CFPB and the dev/design team. 

But, I agree. Overall it's a lot of content for that size.

**Comment by Dnpizarro** [comment]
@mebates will jump on this task for March Sprint.

**Comment by mebates** [comment]
Here's what the page would look like with the introduction block of copy at the H4 specs (18 px, Avenir Next Medium):

![screen shot 2015-03-16 at 9 27 39 am](https://cloud.githubusercontent.com/assets/6223901/6666901/d9666062-cbbe-11e4-83df-336054d7b555.png)

I think that we would also need to fix the gutter between the text and the illustration; the gap is too wide right now.

**Comment by mebates** [comment]
@Scotchester Do you know why that gap appeared to get bigger? I was just playing around in Chrome's developer tools.

**Comment by Dnpizarro** [comment]
@mebates I would agree, the spacing is too wide. Are you getting the same results in other browsers?

**Comment by Scotchester** [comment]
This happens because we have set a max-width on paragraphs of 35em, to keep measure widths to readable lengths. I would recommend not changing that; perhaps we can increase/change the illustration?

Might be cool to pull in some of the stuff @duelj did for the Beta hero graphic.

**Comment by mebates** [comment]
Okay I'll take a look at alternate hero illustration and/or type options!

**Comment by mebates** [comment]
Hey @Scotchester - we don't have that 35 em guideline in the Design Manual. (Web typesetting > Line length) Should it be there? Should it be that strict? 

Also, how does that guideline relate to the grid? (Not sure what kind of grid DM is built on as compared to other web products, just asking for clarification)

**Comment by Scotchester** [comment]
It's largely based on numbers found here: http://webtypography.net/2.1.2

Yes, it probably is worth mentioning in that section as the recommended way to set it in CSS. I do think it should be strict (though if the average character width is 0.5em, it could be bumped up to 36 or even 37, to set the max width closer to the recommended max of 75 characters), because it's a pretty long-held rule of typesetting. It doesn't relate to the grid directly at all. It just constrains the width of paragraphs in situations where the available width in the layout is greater than 35em. (Which is 35 \* the value of `font-size`.)

**Comment by mebates** [comment]
I've made no progress on this! Sorry @Dnpizarro

**Comment by Dnpizarro** [comment]
@mebates do you think we can make progress on this issue for the June Sprint? We can backlog this as well if that works.

**Comment by mebates** [comment]
Making progress on this one :open_mouth: ! Using some elements from the Beta illustration. 

All intro type at 18 px, white background:
![screen shot 2015-06-24 at 3 27 34 pm](https://cloud.githubusercontent.com/assets/6223901/8339474/c0ffd1b4-1a85-11e5-9bf9-f88733c90d89.png)

All intro type at 18 px, teal background (really tall hero, for us, with way too much text):
![screen shot 2015-06-24 at 3 26 34 pm](https://cloud.githubusercontent.com/assets/6223901/8339477/c5d94f6c-1a85-11e5-9a3c-e94eaeee6ef6.png)

Standard 350 px hero with first graph at 22 px. Second paragraph below here but I'm not quite sure what to do with it:
![screen shot 2015-06-24 at 3 26 54 pm](https://cloud.githubusercontent.com/assets/6223901/8339481/ccf97f24-1a85-11e5-87f5-91a090f63b13.png)

**Comment by KimberlyMunoz** [comment]
Hey @mebates!

What was the final decision on this? We have some really content-heavy pages that would be affected by this and want to know if we can implement it.

Thanks!

**Comment by elizbond** [comment]
Feedback from Working Session

Intro before the "The Design Manual is an open-source..."
Sidebar highlighting the information on how to engage

**Comment by duelj** [comment]
##### Thoughts during our working session (7/8/2015)

I had some thoughts around how the homepage might be rearranged if it were a landing page in the FJ mould. It would be nice to feature the three main sections of the design manual on the home page, especially if the hero is just an introduction, just an additional thought, I know it may not be feasible because it would require additional content. 
![fullsizerender](https://cloud.githubusercontent.com/assets/6873734/8580382/6337d6ae-2588-11e5-9674-1cdeb18a87bb.jpg)

**Comment by mebates** [comment]
Hey all - Here's an update to the Design Manual homepage design per our discussion last week, and @duelj's suggestion above. I removed our 'inspiration' section because I feel like it's kind of irrelevant now. The 'news' module would be sometime in the future. 

![screen shot 2015-07-15 at 11 49 51 am](https://cloud.githubusercontent.com/assets/6223901/8702726/a65b8fe8-2ae7-11e5-8031-c5d827ddcbae.png)

**Comment by nataliafitzgerald** [comment]
This looks good! 

Comments/questions/feedback:
- Can you narrow the width of the body text? The line lengths are feeling too long. Same feedback on the intro text block "The Design Manual is an open-source...".
- The space under the "What it includes" and Get involved" headers is too big. It looks perfectly centered between the paragraph before and the paragraph after. This is visually confusing. Not sure if this is the way it's coded or just how it's set up in your ai file.
- The space after the footer bar headers is too big as well "open source" and "a note about our standards".

**Comment by Scotchester** [comment]
Agree with @nataliafitzgerald on the measure width :)

The "What it includes" and "Get involved" headers should have 60px top margin, I think. That would solve Natalia's second point.

The footer header spacing is an existing bug, I think. I can take care of that in the course of this update.

I'd like to see another take with a white background in the hero area. I don't totally love the idea of using a tint of one of our secondary colors as the main use of color on the website created to showcase our branding, know what I mean? Might be worth looking at the two lighter CFPB Greens, too.

**Comment by nataliafitzgerald** [comment]
@Scotchester - As beautiful as the light teal tint looks, I support trying out white and the lighter greens to see if they may work. Brand always comes first.

**Comment by mebates** [comment]
I think the white is going to be tricky with the left gray sidebar. I like the light green and fixed the spacing:

![screen shot 2015-07-16 at 1 44 27 pm](https://cloud.githubusercontent.com/assets/6223901/8730896/c87c2170-2bc1-11e5-8e99-9e8083224a90.png)

**Comment by Scotchester** [comment]
Looks awesome! Let's roll with that.

**Comment by mebates** [comment]
I probably need to tweak the hero a bit to match the new hero guidelines.

**Comment by nataliafitzgerald** [comment]
@mebates - As you work to update the hero, we will need to cut down the subheading content so that it is no more than 3 lines at desktop or 185 maximum character count (including spaces). https://cfpb.github.io/design-manual/ui-toolkit/heroes.html

**Comment by keelerr** [comment]
I noticed the content was rewritten sometime last year... Is this what we're going forward with? Is there a desire to change anything else on the page beyond the hero?

**Comment by kurzn** [comment]
We will need to update the "what it includes" content to match the new IA once that is in place. What is the timeframe for getting the new homepage up? I'm guessing it will be ready before the new IA. So the rest of the page could get updated content, and then the "what it includes" can be updated once the new IA is pushed (assuming we want to keep that section at all). 

I'm happy to take a stab at new content if need be.  

Thoughts?

**Comment by mebates** [comment]
Here's the latest, with the hero updated to match standards. I tweaked the text slightly to get it to 185 characters. @keelerr and @kurzn I'm not sure if you're requesting further content changes from here. This should launch before IA changes, so let's just plan to update it after you make those.

![screen shot 2016-02-23 at 10 47 51 am](https://cloud.githubusercontent.com/assets/6223901/13257043/29387d62-da1b-11e5-84bd-1d666752f4dc.png)

One other thing I think we should update is in the footer; we give old examples. Can we point to Owning a Home, Planning for Retirement and eRegs?
![screen shot 2016-02-23 at 10 47 39 am](https://cloud.githubusercontent.com/assets/6223901/13257045/2cbaadc0-da1b-11e5-9a9c-1ca0ac1366bb.png)

**Comment by kurzn** [comment]
@mebates  we can update it all at once after new IA push. Seems like the fastest option to get this up.

**Comment by kurzn** [comment]
@caheberer is this still in the cue to get updated?

**Comment by caheberer** [comment]
Yes, @Scotchester still has it (and far too many other things) on his plate.

**Comment by kurzn** [comment]
Roger that 👍

**Comment by stephanieosan** [comment]
Can we get a @Scotchester update on this one? Is this in progress, not started? Abandoned?

**Comment by jimmynotjim** [comment]
@Scotchester do you want me to pick this up?

**Comment by Scotchester** [comment]
@jimmynotjim, if you've got the time and inclination, feel free!

Honestly, I barely got started on it. No work worth handing off, really.

**Comment by jimmynotjim** [comment]
Realizing this isn't worth working on till the Manual has been updated to CFv4 because all the things included are going to change. Putting on hold while I attempt to get that out.
